src/Endpoints/class-base-endpoint.php

```diff
@@ -126,7 +129,7 @@ public function register_endpoint(
 		string $callback,
 		array $methods = array( 'GET' )
 	): void {
-		if ( ! apply_filters( 'wp_parsely_enable_' . convert_endpoint_to_filter_key( $endpoint ) . '_api_proxy', true ) ) {
+		if ( ! apply_filters( 'wp_parsely_enable_' . convert_endpoint_to_filter_key( $endpoint ) . '_endpoint', true ) ) {
```

Comment:
This change is correct and I don't think anyone uses this filter. Having said that, if anyone does it would be a breaking change for them, potentially giving access to unauthorized endpoints. I think we should actually deprecate that filter and completely replace it with your suggestion in a BC release later.

This is of course outside of the current PR's scope. For now, I'd just revert that to its original value and file this so we don't lose track of it.

Thoughts?

```suggestion
		if ( ! apply_filters( 'wp_parsely_enable_' . convert_endpoint_to_filter_key( $endpoint ) . '_api_proxy', true ) ) {
```